tidy3d/components/monitor.py

```diff
@@ -456,6 +456,11 @@ def storage_size(self, num_cells: int, tmesh: ArrayFloat1D) -> int:
 
 class ModeMonitor(AbstractModeMonitor):
     """:class:`Monitor` that records amplitudes from modal decomposition of fields on plane.
+    The amplitudes are defined as
+    ``recorded_field.dot(mode_solver_data) / mode_solver_data.dot(mode_solver_data)``, where
```

Comment:
it's actually `mode_solver_data.dot(recorded_field) / mode_solver_data.dot(mode_solver_data)` but I don't think it matters

Reply:
Thanks, I'll change anyway.